manage.py

```diff
@@ -3,6 +3,10 @@
 import sys
 
 if __name__ == "__main__":
+    # Add argument needed (and easily forgotten) when using Vagrant.
+    if 'runserver' in sys.argv and len(sys.argv) == 2:
```

Comment:
Thanks for the patch. I think this is a bad practice and shouldn't be in manage.py.